# SR.CAPS_STATE > SR.MS_CAPS_STATE
# (c) zet
Index: src\Client\ConfigForm.java
===================================================================
--- ConfigForm.java	(revision 540)
+++ ConfigForm.java	(working copy)
@@ -117,7 +117,7 @@
         message.append(SR.MS_STORE_PRESENCE,null);        
         message.append(SR.MS_COMPOSING_EVENTS, null);
 //#if (!MIDP1)
-        message.append(SR.CAPS_STATE, null);
+        message.append(SR.MS_CAPS_STATE, null);
 //#endif
         
         boolean mv[]={
