# разрешение локализации пункта File Transfers в меню Tools
# (c) zet <shelukhin@mail.ru>
Index: src/Client/RosterToolsMenu.java
===================================================================
--- RosterToolsMenu.java	(revision 1156)
+++ RosterToolsMenu.java	(working copy)
@@ -54,7 +54,7 @@
         
 //#if (FILE_IO && FILE_TRANSFER)
         addItem("root",4);
-        addItem("File Transfers", 5);
+        addItem(SR.MS_FILE_TRANSFERS, 5);
 //#endif
         
         
