# Как вариант (для Nokia 208x208 сдвиг 33 достаточен)
# (c) zet
Index: src\Conference\affiliation\ConferenceQuickPrivelegeModify.java
===================================================================
--- ConferenceQuickPrivelegeModify.java	(revision 925)
+++ ConferenceQuickPrivelegeModify.java	(working copy)
@@ -22,6 +22,7 @@
 import javax.microedition.lcdui.Form;
 import javax.microedition.lcdui.StringItem;
 import javax.microedition.lcdui.TextField;
+import javax.microedition.lcdui.Spacer;
 import locale.SR;
 import ui.controls.TextFieldCombo;
 
@@ -87,6 +88,8 @@
         }
         f.append(new StringItem("User",user.toString()));
         
+        f.append(new Spacer(256, 33));
+        
         reason=new TextFieldCombo("Reason", "", 64, TextField.ANY, "reason", display);
         f.append(reason);
         
