# 500 -> 800
# (c) zet
Index: src\Client\MessageEdit.java
===================================================================
--- MessageEdit.java	(revision 574)
+++ MessageEdit.java	(working copy)
@@ -49,7 +49,7 @@
         this.to=to;
         this.display=display;
         parentView=display.getCurrent();
-        t=new TextBox(to.toString(),null,500, TextField.ANY);
+        t=new TextBox(to.toString(),null,800, TextField.ANY);
         
         try {
             if (body!=null) t.setString(body);
