# Подготовка к локализации
# (c) zet
Index: src\locale\SR.java
===================================================================
--- SR.java	(revision 853)
+++ SR.java	(working copy)
@@ -227,7 +227,7 @@
     public   static String MS_INVITE=loadString("Invite to conference");
     public   static String MS_INVITE_REASON=loadString("Reason");
     public   static String MS_YOU_HAVE_BEEN_INVITED=loadString("You have been invited to ");
-    public final static String MS_SURE_CLEAR="Are You sure want to clear messagelist?";
+    public   static String MS_SURE_CLEAR=loadString("Are You sure want to clear messagelist?");
     public   static String MS_DISCO_ROOM=loadString("Participants");
     public   static String MS_CAPS_STATE=loadString("Abc");
     
@@ -244,37 +244,37 @@
     
     public   static String MS_TOKEN=loadString("Google token request");
     
-    public final static String MS_SASL="SASL login";
-    public final static String MS_FEATURES="Features";
-    public final static String MS_SHOWPWD="Show password";
-    public final static String MS_NO_VERSION_AVAILABLE="No client version available";
-    public final static String MS_MSG_LIMIT="Message limit";
+    public   static String MS_SASL=loadString("SASL login");
+    public   static String MS_FEATURES=loadString("Features");
+    public   static String MS_SHOWPWD=loadString("Show password");
+    public   static String MS_NO_VERSION_AVAILABLE=loadString("No client version available");
+    public   static String MS_MSG_LIMIT=loadString("Message limit");
     
-    public final static String MS_OPENING_STREAM="Opening stream";
-    public final static String MS_SASL_STREAM="SASL handshake";
-    public final static String MS_ZLIB="Using compression";
-    public final static String MS_AUTH="Authenticating";
-    public final static String MS_RESOURCE_BINDING="Resource binding";
-    public final static String MS_SESSION="Initiating session";
+    public   static String MS_OPENING_STREAM=loadString("Opening stream");
+    public   static String MS_SASL_STREAM=loadString("SASL handshake");
+    public   static String MS_ZLIB=loadString("Using compression");
+    public   static String MS_AUTH=loadString("Authenticating");
+    public   static String MS_RESOURCE_BINDING=loadString("Resource binding");
+    public   static String MS_SESSION=loadString("Initiating session");
     
-    public final static String MS_TEXTWRAP="Text wrapping";
-    public final static String MS_TEXTWRAP_CHARACTER="by chars";
-    public final static String MS_TEXTWRAP_WORD="by words";
+    public   static String MS_TEXTWRAP=loadString("Text wrapping");
+    public   static String MS_TEXTWRAP_CHARACTER=loadString("by chars");
+    public   static String MS_TEXTWRAP_WORD=loadString("by words");
 
-    public final static String MS_INFO="Info";
-    public final static String MS_REPLY="Reply";
-    public final static String MS_DIRECT_PRESENCE="Send status";
+    public   static String MS_INFO=loadString("Info");
+    public   static String MS_REPLY=loadString("Reply");
+    public   static String MS_DIRECT_PRESENCE=loadString("Send status");
 
-    public final static String MS_CONFIRM_BAN="Are you sure want to BAN this person?";
-    public final static String MS_NO_REASON="No reason";
+    public   static String MS_CONFIRM_BAN=loadString("Are you sure want to BAN this person?");
+    public   static String MS_NO_REASON=loadString("No reason");
 
-    public final static String MS_RECENT="Recent";
-    public final static String MS_CAMERASHOT="Shot";
+    public   static String MS_RECENT=loadString("Recent");
+    public   static String MS_CAMERASHOT=loadString("Shot");
 
-    public final static String MS_SELECT_FILE="Select file";
-    public final static String MS_LOAD_PHOTO="Load Photo";
-    public final static String MS_CLEAR_PHOTO="Clear Photo";
-    public final static String MS_CAMERA="Camera";
+    public   static String MS_SELECT_FILE=loadString("Select file");
+    public   static String MS_LOAD_PHOTO=loadString("Load Photo");
+    public   static String MS_CLEAR_PHOTO=loadString("Clear Photo");
+    public   static String MS_CAMERA=loadString("Camera");
 
     public static String MS_XMLLANG;
     public static String MS_IFACELANG;
