# устранение несовпадений
# (c) zet
Index: src\Info\Version.java
===================================================================
--- Version.java	(revision 843)
+++ Version.java	(working copy)
@@ -20,7 +20,7 @@
 //#if (!ZLIB)
 //#     public final static String version="$BOMBUSVERSION$";
 //#else
-    public final static String version="$BOMBUSVERSION$-Zlib";
+    public final static String version="$BOMBUSVERSION$-ZLib";
 //#endif
     // this string will be patched by build.xml/post-preprocess
     
