# "Color theme" и прочее
# (c) zet <zet@bombus-im.org>
Index: src/locale/SR.java
===================================================================
--- SR.java	(revision 1226)
+++ SR.java	(working copy)
@@ -199,9 +199,9 @@
     public   static String MS_SUSPEND = loadString( "Suspend" );
     public   static String MS_ALERT_PROFILE_CMD = loadString( "Alert Profile >" );
     public   static String MS_MY_VCARD = loadString( "My vCard" );
-    public   static String MS_TRANSPORTS = loadString( "transports" );
+    public   static String MS_TRANSPORTS = loadString( "Transports" );
     public   static String MS_NEW_ACCOUNT = loadString( "New Account" );
-    public   static String MS_SELF_CONTACT = loadString( "self-contact" );
+    public   static String MS_SELF_CONTACT = loadString( "Self-contact" );
     public   static String MS_VCARD = loadString( "vCard" );
     public   static String MS_SET_SUBJECT = loadString( "Set Subject" );
     public   static String MS_TOOLS = loadString( "Tools" );
@@ -352,6 +352,7 @@
 //2007-11-07
     public static String MS_SEARCH=loadString("Search");
     public static String MS_REGISTER=loadString("Register");
+    public static String MS_COLOR_THEME=loadString("Color theme");
     
     
     public static String MS_XMLLANG;
