# группы свёрнуты по умолчанию при загрузке ростера
# (c) zet
Index: src\Client\Group.java
===================================================================
--- Group.java	(revision 843)
+++ Group.java	(working copy)
@@ -32,7 +32,7 @@
     private int tncontacts;
     public int unreadMessages=0;
     
-    boolean collapsed;
+    boolean collapsed=true;
     
     public Group(String name /*, String label*/) {
         super(RosterIcons.getInstance());
