# "Cancel" -> "Back"
# (c) zet <shelukhin@mail.ru>
Index: src/Client/ActiveContacts.java
===================================================================
--- ActiveContacts.java	(revision 1147)
+++ ActiveContacts.java	(working copy)
@@ -49,7 +49,7 @@
     
     Vector activeContacts;
     
-    private Command cmdCancel=new Command(SR.MS_CANCEL, Command.BACK, 99);
+    private Command cmdCancel=new Command(SR.MS_BACK, Command.BACK, 99);
     private Command cmdSelect=new Command(SR.MS_SELECT, Command.SCREEN, 1);
     /** Creates a new instance of ActiveContacts */
     public ActiveContacts(Display display, Contact current) {
