# "Cancel" -> "Back"
# (c) zet <shelukhin@mail.ru>
Index: src/Client/ConfigForm.java
===================================================================
--- ConfigForm.java	(revision 1147)
+++ ConfigForm.java	(working copy)
@@ -99,7 +99,7 @@
     Command cmdOk=new Command(SR.MS_OK,Command.OK,1);
     //Command cmdSign=new Command("- (Sign)",Command.ITEM,2);
     Command cmdPlaySound=new Command(SR.MS_TEST_SOUND, Command.ITEM,10);
-    Command cmdCancel=new Command(SR.MS_CANCEL, Command.BACK,99);
+    Command cmdCancel=new Command(SR.MS_BACK, Command.BACK,99);
     
     Config cf;
     boolean ra[];
