# разрешение локализации профилей сигнализации
# (c) zet <shelukhin@mail.ru>
Index: src/Client/AlertProfile.java
===================================================================
--- AlertProfile.java	(revision 1156)
+++ AlertProfile.java	(working copy)
@@ -45,7 +45,7 @@
     public final static int NONE=4;
     
     private final static String[] alertNames=
-    { "Auto", "All signals", "Vibra", "Sound", "No signals"};
+    { SR.MS_ALERT_PROFILE_AUTO, SR.MS_ALERT_PROFILE_ALLSIGNALS, SR.MS_ALERT_PROFILE_VIBRA, SR.MS_ALERT_PROFILE_SOUND, SR.MS_ALERT_PROFILE_NOSIGNALS };
     
     private Profile profile=new Profile();
     int defp;
@@ -87,7 +87,7 @@
         public int getImageIndex(){return index+RosterIcons.ICON_PROFILE_INDEX;}
         public String toString(){ 
             StringBuffer s=new StringBuffer(alertNames[index]);
-            if (index==defp) s.append(" (default)");
+            if (index==defp) s.append(SR.MS_ALERT_PROFILE_DEFAULT);
             return s.toString();
         }
     }
