# увеличение количества хранимых записей MRU
# (c) zet <shelukhin@mail.ru>
Index: src/ui/controls/TextFieldCombo.java
===================================================================
--- TextFieldCombo.java	(revision 1156)
+++ TextFieldCombo.java	(working copy)
@@ -87,7 +87,7 @@
         int i=0;
         if (result.length()==0) return result;
         while (i<recentList.size()) {
-            if ( result.equals((String)recentList.elementAt(i)) || i>9 ) recentList.removeElementAt(i);
+            if ( result.equals((String)recentList.elementAt(i)) || i>99 ) recentList.removeElementAt(i);
             else i++;
         }
         recentList.insertElementAt(result, 0);
