# разрешение локализации некоторых итемов
# (c) zet <zet@bombus-im.org>
Index: src/Info/InfoWindow.java
===================================================================
--- InfoWindow.java	(revision 1226)
+++ InfoWindow.java	(working copy)
@@ -43,7 +43,7 @@
     private Form form;
 
     private Command cmdBack=new Command(SR.MS_CLOSE, Command.BACK, 99);
-    private Command cmdUpgrade=new Command("Versions", Command.SCREEN, 1);
+    private Command cmdUpgrade=new Command(SR.MS_VERSIONS, Command.SCREEN, 1);
     /** Creates a new instance of InfoWindow */
     public InfoWindow(Display display) {
         this.display=display;
@@ -66,13 +66,16 @@
 //#endif
                 ));
         
-        StringBuffer memInfo=new StringBuffer("\n\nMemory:\n");
-        memInfo.append("Free=");
+        StringBuffer memInfo=new StringBuffer("\n\n");
+        memInfo.append(SR.MS_MEMORY); 
+        memInfo.append("\n");
+        memInfo.append(SR.MS_MEMORY_FREE);
         //mem.append(Runtime.getRuntime().freeMemory()>>10);
         //mem.append("\nFree=");
         System.gc();
         memInfo.append(Runtime.getRuntime().freeMemory()>>10);
-        memInfo.append("\nTotal=");
+        memInfo.append("\n");
+        memInfo.append(SR.MS_MEMORY_TOTAL);
         memInfo.append(Runtime.getRuntime().totalMemory()>>10);
         form.append(memInfo.toString());
 //#if ZLIB
@@ -81,7 +84,7 @@
         try {
             conn_stats=StaticData.getInstance().roster.theStream.getStreamStats();
         } catch (Exception e) {
-            conn_stats="Disconnected";
+            conn_stats=SR.MS_DISCONNECTED;
         }
         form.append(conn_stats);
 //#endif
Index: Upgrade.java
===================================================================
--- Upgrade.java	(revision 1226)
+++ Upgrade.java	(working copy)
@@ -38,6 +38,7 @@
 import javax.microedition.lcdui.Displayable;
 import javax.microedition.lcdui.List;
 import midlet.Bombus;
+import locale.SR;
 
 /**
  *
@@ -45,8 +46,8 @@
  */
 public class Upgrade implements Runnable, CommandListener{
    
-    private Command cmdBack=new Command("Back", Command.BACK, 99);
-    private Command cmdInstall=new Command("Install", Command.ITEM, 1);
+    private Command cmdBack=new Command(SR.MS_BACK, Command.BACK, 99);
+    private Command cmdInstall=new Command(SR.MS_INSTALL, Command.ITEM, 1);
     private final static String VERSION_URL=Version.BOMBUS_SITE_URL+"/version.php";
     List list;
     Vector versions[];
@@ -58,7 +59,7 @@
     public Upgrade(Display display) {
         this.display=display;
         parentView=display.getCurrent();
-        list=new List("Available versions", List.IMPLICIT);
+        list=new List(SR.MS_AVAILABLE_VERSIONS, List.IMPLICIT);
         list.setCommandListener(this);
         list.addCommand(cmdBack);
         display.setCurrent(list);
