# ВСЕ группы свёрнуты по умолчанию
# (c) zet
Index: src/Conference/ConferenceGroup.java
===================================================================
--- ConferenceGroup.java	(revision 1217)
+++ ConferenceGroup.java	(working copy)
@@ -45,6 +45,9 @@
 	super(name);
 	this.label=label;
 	imageExpandedIndex=RosterIcons.ICON_GCJOIN_INDEX;
+
+    onSelect();
+    
         this.type=Groups.TYPE_MUC;
     }
# убран recent для поля room в форме join conference
# (c) zet
Index: src/Conference/ConferenceForm.java
===================================================================
--- ConferenceForm.java	(revision 1217)
+++ ConferenceForm.java	(working copy)
@@ -32,6 +32,7 @@
 import locale.SR;
 import ui.*;
 import javax.microedition.lcdui.*;
+import javax.microedition.lcdui.Spacer;
 import ui.controls.NumberField;
 import ui.controls.TextFieldCombo;
 
@@ -99,8 +100,10 @@
         parentView=display.getCurrent();
         
         Form formJoin=new Form(SR.MS_JOIN_CONFERENCE);
+    
+       // formJoin.append(new Spacer(256, 33));
         
-        roomField=new TextFieldCombo(SR.MS_ROOM, room, 64, ConstMIDP.TEXTFIELD_URL, "muc-room", display);
+        roomField=new TextField(SR.MS_ROOM, room, 64, ConstMIDP.TEXTFIELD_URL);
         TextFieldCombo.setLowerCaseLatin(roomField); 
         formJoin.append(roomField);
         
